wcomponents-theme/src/test/intern/wc.dom.shed.test.js

```diff
@@ -534,6 +534,36 @@ define(["intern!object", "intern/chai!assert", "intern/resources/test.utils!"],
 				var test = document.getElementById("visibletests1");
 				assert.isFalse(controller.isHidden(test));
 			},
+			testIsHiddenTextNodeChildOfElement: function() {
+				var parent = document.createElement("span");
+				var child = parent.appendChild(document.createTextNode("tn"));
+				assert.isFalse(controller.isHidden(parent));
+				assert.isFalse(controller.isHidden(child));
+				controller.hide(parent);
+				assert.isTrue(controller.isHidden(parent));
+				assert.isTrue(controller.isHidden(child));
+			},
+			testIsHiddenCommentNodeChildOfElement: function() {
+				var parent = document.createElement("span");
+				var child = parent.appendChild(document.createComment("comment"));
+				assert.isFalse(controller.isHidden(parent));
+				assert.isFalse(controller.isHidden(child));
+				controller.hide(parent);
+				assert.isTrue(controller.isHidden(parent));
+				assert.isFalse(controller.isHidden(child));
```

Comment:
Interesting, not really sure what the right thing to do is here. You think comment nodes are essentially not able to be hidden? Makes no sense either way I guess.

Reply:
Comment nodes are not elements, they do not have attributes. So they cannot be hidden.

Reply:
Yes, remove the test, we'll leave the behavior undefined.
Alternatively simply assert that an exception is not thrown, maybe this `assert.doesNotThrow`